src/components/shapesdemos.tsx

```diff
@@ -55,6 +55,14 @@ const polygoncardExample = `
 />
 `;
 
+const triangleExample = `
+<Triangle
+    size="150px"
+    zIndex={2}
+    color="linear-gradient(135deg, #f9a8d4, #eab309)"
```

Comment:
```suggestion
    color="linear-gradient(135deg, #fde047, #eab308)"
```

We are following [tailwind colors](https://tailwindcss.com/docs/customizing-colors) so it is good to stay consistent with the gradient theme which we have used in other components.